src/common/checksums.cpp

```diff
@@ -328,7 +328,7 @@ ComputeChecksum *ValidateChecksumHeader::prepareStart(const QByteArray &checksum
 
     if (!parseChecksumHeader(checksumHeader, &_expectedChecksumType, &_expectedChecksum)) {
         qCWarning(lcChecksums) << "Checksum header malformed:" << checksumHeader;
-        emit validationFailed(tr("The checksum header is malformed."));
+        emit validationFailed(tr("The checksum header is malformed."), QByteArray(), QByteArray(), QString());
```

Comment:
> why not put _filePath in the third arg ?
@mgallien 
Why do we need it there?

Reply:
the question for is really why the last three args are optional ?
are we sure to be able to indicate where the error is happening (i.e. the filename) ?

Reply:
> the question for is really why the last three args are optional ?
> are we sure to be able to indicate where the error is happening (i.e. the filename) ?

@mgallien The filename is optional, because, we only need it in one case. I'd prefer not to add it to every emit to `validattionFailed` up until we start to need it in other cases. If you insist, I can add it to other cases, but, really, it is not needed for every case but one.

Reply:
I guess I understand even if I am also misled by the many log statements that have somehow missing information that I am under the impression that I would miss data if I had to analyze some logs

Reply:
@mgallien The problem is, we don't have logs for validateFailed slots yet. So, me, adding `_filePath` to every emit, would just bring unused information. Also, I remember you've been mentioning that we don't want too much logs because logger can cause a crash if we log too much text, but, I am going to add this argument into this PR just to make it move forward.